Raven.Abstractions/Smuggler/SmugglerDatabaseApiBase.cs

```diff
@@ -1014,6 +1014,7 @@ public static void WriteLastEtagsToFile(OperationState result, string backupPath
 
             exportSectionRegistar.Add(Constants.BuildVersion, async () =>
             {
+                Operations.ShowProgress("Skipping import of indexes");
```

Comment:
You need to put exact reason here. Maybe "Detected 4.x file format. Automatically disabled index and transformer import." ?